From 6d690ebf64ce640afce3085f01da8694a04853a9 Mon Sep 17 00:00:00 2001
From: Alex Kiernan <alex.kiernan@gmail.com>
Date: Tue, 20 Jun 2023 19:24:04 +0100
Subject: [PATCH] lib/deploy: Use off_t not __off_t

Not clear why this was __off_t which is the sole appearance in the code
base, but it breaks musl builds.

Upstream-Status: Submitted [https://github.com/ostreedev/ostree/pull/2896]
Signed-off-by: Alex Kiernan <alex.kiernan@gmail.com>
---
 src/libostree/ostree-sysroot-deploy.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libostree/ostree-sysroot-deploy.c b/src/libostree/ostree-sysroot-deploy.c
index 8fcd5e802758..2454a5877d94 100644
--- a/src/libostree/ostree-sysroot-deploy.c
+++ b/src/libostree/ostree-sysroot-deploy.c
@@ -2536,7 +2536,7 @@ get_kernel_layout_size (OstreeSysroot *self, OstreeDeployment *deployment, guint
 /* This is a roundabout but more trustworthy way of doing a space check than
  * relying on statvfs's f_bfree when you know the size of the objects. */
 static gboolean
-dfd_fallocate_check (int dfd, __off_t len, gboolean *out_passed, GError **error)
+dfd_fallocate_check (int dfd, off_t len, gboolean *out_passed, GError **error)
 {
   /* If the requested size is 0 then return early. Passing a 0 len to
    * fallocate results in EINVAL */
